File: 4/3.c
#include <stdio.h>

int main()
{
	int a, b, c;

	printf(" hello , enter 3 numbers interger :\n");
	scanf("%d %d %d", &a, &b, &c);
	if ( a >= b &&  a >= c)
	{
		printf("%d is greatest\n", a);
	}
	else if ( b >= a && b >= c)
	{
		printf("%d is greatest\n", b);
	}
	else if ( c >= a && c >= b)
	{
		printf("%d is greatest\n", c);
	}
	getchar();
	return (0);
}

File: 4/3.py
#!/usr/bin/python3

print("hello enter 3 number interger :")

a = int(input("enter number 1 :"))
b = int(input("enter number 2 :"))
c = int(input("enter number 3 :"))

if a >= b and a >= c :
		print(f"{a} is greatest\n")
elif b >= a and  b >= c:
		print(f"{b} is greatest\n")
elif  c >= a and c >= b:
		print(f"{c} is greatest\n")
